Makefile

```diff
@@ -235,12 +230,12 @@ setup:
 	$(MAKE) -C tools
 	python3 tools/fixbaserom.py
 	python3 tools/extract_baserom.py
-	python3 extract_assets.py -t $(N_THREADS)
+	python3 extract_assets.py
 
 ## Assembly generation
 disasm:
 	$(RM) -rf asm data
-	python3 tools/disasm/disasm.py -j $(N_THREADS)
+	python3 tools/disasm/disasm.py -j ${N_THREADS}
```

Comment:
What's the reason to change `$()` to `${}`?

Reply:
Looks like a copypaste error. I'll fix it and make sure it still works.